src/Components/Program/Program.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import { Button, Container, Collapse, Image, Row, Col } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faDownload,
+  faGlobe,
+  faEnvelope,
+  faPhoneAlt,
+  faMapMarkerAlt,
+  faChevronDown,
+  faChevronUp,
+} from "@fortawesome/free-solid-svg-icons";
+
+export const Program = () => {
+  const program = {
+    id: 1,
+    title: "Nursing Assistant Program",
+    location: "Seatle Central College",
+    educationLevel: "College | Academy",
+    careerEmphasis: "Nursing",
+    description:
+      "The nursing program at Seatle Central offers an Associate in Nursing Direct Transfer Degree/Major Related Program (AN DTA/MRP) with...",
+    image:
+      "https://www.freevector.com/uploads/vector/preview/30485/Healthcare_characters_vector_5-01.jpg",
+    elegibilityRequirements: ["one", "two"],
+    targetPopulation: ["one", "two"],
+    other: ["one", "two"],
+  };
+  const [contactInfo, setContactInfo] = useState(false);
+  const [programDetail, setProgramDetail] = useState(false);
+
+  return (
+    <Container className="mb-4">
+      <h1 className="text-primary text-center">{program.title}</h1>
+      <Row className="align-items-center">
+        <Col xs={4}>
+          <Image src={program.image} thumbnail />
+        </Col>
+        <Col>
+          <p>{program.location}</p>
+          <p>{program.educationLevel}</p>
+          <p>{program.careerEmphasis}</p>
+        </Col>
+      </Row>
+      <p className="mt-4">{program.description}</p>
+      <div className="my-4">
+        <a href="#">Show more</a>
```

Comment:
Can you add overflow property to program description above and make it so that clicking on "Show more" reveals the whole text?